server/data/quotes.json

```diff
@@ -220,7 +200,17 @@
     "author": "George Crane"
   },
   {
-    "sentence": "The more languages you know, the more you are human.", 
+    "sentence": "The more languages you know, the more you are human.",
     "author": " Tomáš Garrigue Masaryk"
-  }
+  },
+ {
+     "sentence": "Language is not a genetic gift, it is a social gift. Learning a new language is becoming a member of the club – the community of speakers of that language.",
+     "author": " Frank Smith"
+ },
+ {
+     "sentence": "A special kind of beauty exists which is born in language, of language, and for language.",
+     "author": "Gaston Bachelard"
+ }
+
```

Comment:
Please remove the new lines here so it’s cleaner!

Reply:
Which new line?

Reply:
Lines 214 and 215

Reply:
Done!!